nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestWorkManagerObject.java

```diff
@@ -0,0 +1,86 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour Al Kotob
+ */
+
+package org.nuxeo.rest.management;
+
+import static org.junit.Assert.assertEquals;
+import static org.nuxeo.rest.management.work.manager.WorkManagerObject.TIMEOUT_SECONDS_PARAM_KEY;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.automation.test.AutomationFeature;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.LogCaptureFeature;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.sun.jersey.core.util.MultivaluedMapImpl;
+
+/**
+ * @since 11.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features({ AutomationFeature.class, LogCaptureFeature.class })
+@Deploy("org.nuxeo.rest.management:OSGI-INF/test-work-dead-letter-queue.xml")
+public class TestWorkManagerObject extends ManagementBaseTest {
+
+    private static final String SUCCESS_KEY = "success";
+
+    private static final String TOTAL_KEY = "total";
+
+    public static final String PATH = "site/management/workmanager/runworksinfailure";
+
+    public static final String OK = "ok";
+
+    @Inject
+    LogCaptureFeature.Result logCaptureResult;
+
+    @Test
+    @LogCaptureFeature.FilterOn(logLevel = "WARN")
+    public void testRunningStatus() throws IOException {
+        // Test with timeout parameter
+        MultivaluedMap<String, String> formData = new MultivaluedMapImpl();
+        formData.add(TIMEOUT_SECONDS_PARAM_KEY, "10");
+        try (CloseableClientResponse response = httpClientRule.post(PATH, formData)) {
+            assertEquals(HttpServletResponse.SC_OK, response.getStatus());
+            JsonNode result = mapper.readTree(response.getEntityInputStream());
+            assertEquals(0, result.get(TOTAL_KEY).asInt());
+            assertEquals(0, result.get(SUCCESS_KEY).asInt());
+        }
+
+        // Test without timeout parameter
+        int warnCountBefore = logCaptureResult.getCaughtEventMessages().size();
+        try (CloseableClientResponse response = httpClientRule.post(PATH, null)) {
+            assertEquals(HttpServletResponse.SC_OK, response.getStatus());
+            JsonNode result = mapper.readTree(response.getEntityInputStream());
+            assertEquals(0, result.get(TOTAL_KEY).asInt());
+            assertEquals(0, result.get(SUCCESS_KEY).asInt());
+        }
+        assertEquals(warnCountBefore, logCaptureResult.getCaughtEventMessages().size());
```

Comment:
What are you testing here? Seems not clear to me.

Reply:
I'm testing I didn't get an extra warn for passing null as params. I use the method signature that doesn't need args. If you pass null here like suggested, you will get a warn.

I leave it to you to decide what to do. 
I just thought it would be cleaner to use the method signature without args when I don't have any, and avoid a useless warn.

Reply:
my suggestion is to split this test into two tests : testing that the operation will run correctly when the timeout is set and a 2nd method to ensure the same thing if we dont provide this time out param